setup.cfg

```
@@ -1,13 +1,83 @@
 [metadata]
 description-file = README.rst
+url = https://github.com/physiopy/peakdet
+download_url = https://github.com/physiopy/peakdet
+author = peakdet developers
+maintainer = Ross Markello, Stefano Moia
+maintainer_email = rossmarkello@gmail.com, s.moia@bcbl.eu
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+license = Apache-2.0
+description = A python toolbox for reproducible physiological data analysis
+long_description = file:README.md
+long_description_content_type = text/markdown; charset=UTF-8
+platforms = OS Independent
+provides =
+    peakdet
+
+[options]
+python_requires = >=3.6.1
+install_requires =
+	Gooey
+    matplotlib >=3.1.1, !=3.3.0rc1
+    numpy >=1.9.3, !=*rc*
+    pillow
+    psutil
+    PyYAML >=5.1, !=*rc*
+    scipy
+    wxpython
+tests_require =
+    pytest >=5.3
+    pytest-cov
+test_suite = pytest
+zip_safe = False
+packages = find:
+include_package_data = True
+
+[options.extras_require]
+duecredit =
+    duecredit
+doc =
+	pandas
+    sphinx >=2.0
+    sphinx-argparse
+    sphinx_rtd_theme
+style =
+    flake8 >=3.7
+    flake8-docstrings >=1.5
+test =
+    pytest >=5.3
+    pytest-cov
+    %(style)s
+all =
+    %(doc)s
+    %(duecredit)s
+    %(style)s
+    %(test)s
 
 [flake8]
-ignore = E402, W503
+exclude=
+    *build/
+    heuristics
+    tests
+ignore = E126, E402, W503
```

```suggestion
ignore = E126, E203, E402, W503
```
To summarize these four: E126 is continuation line over-indented for hanging indent. E203 is whitespace before ':'. E402 is module level import not at top of file. W503 is line break occurred before a binary operator (and is in direct competition with W504, which is line break _after_ binary operator).

`black` recommends ignoring E203 and W503, and even if we don't actively _use_ `black`, I think it's a good idea to match our linter settings to their general criteria, which is why I'd like to add E203.

I get why E402 is included- I've seen cases where there's no way to get around it. I'm a little confused about E126 though. It seems pretty uncontroversial?